libethereum/TransactionQueue.cpp

```diff
@@ -31,12 +31,34 @@ using namespace dev::eth;
 const char* TransactionQueueChannel::name() { return EthCyan "┉┅▶"; }
 const char* TransactionQueueTraceChannel::name() { return EthCyan " ┅▶"; }
 
+TransactionQueue::TransactionQueue()
+{
+	// Allow some room for other activity
+	unsigned verifierThreads = std::max(thread::hardware_concurrency(), 3U) - 2U;
```

Comment:
don't really see any harm. if there's 256 incoming transactions, they'll help and if there's only one, then 5 will never get woken up anyway.